models/balancer/balancer_bpt_prices.sql

```diff
@@ -0,0 +1,32 @@
+{{ config(
+    schema = 'balancer',
+    alias = 'bpt_prices',
+    post_hook='{{ expose_spells(\'["ethereum", "arbitrum", "polygon", "gnosis"]\',
+                            "project",
+                            "balancer",
+                            \'["thetroyharris"]\') }}'
+    )
+}}
+
+{% set balancer_models = [
+    ref('balancer_v2_ethereum_bpt_prices'),
+    ref('balancer_v2_arbitrum_bpt_prices'),
+    ref('balancer_v2_polygon_bpt_prices'),
+    ref('balancer_v2_gnosis_bpt_prices')
```

Comment:
forgot to add optimism?

Reply:
I removed it on purpose. Will check with the team and see if we want to add it in the future. However, I think we are good to move forward without it for now.

Reply:
gotcha!